plone/app/contenttypes/browser/templates/image_view_fullscreen.pt

```diff
@@ -32,6 +38,7 @@
       border: 0;
       display:block;
       margin:0 auto;
+      max-width: 100%; height: auto;
```

Comment:
Would it be possible to keep one rule per line, so that whenever it has to be updated changes are easily to spot?